src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -717,27 +717,22 @@
   "Returns a reducible collection of rows as maps from `db` and a given SQL query. This is similar to [[jdbc/reducible-query]] but reuses the
   driver-specific configuration for the Connection and Statement/PreparedStatement. This is slightly different from [[execute-reducible-query]]
   in that it is not intended to be used as part of middleware. Keywordizes column names. "
-  [db [sql & params]]
```

Comment:
You can't change the signatures of public driver functions like this out of the blue. We need to give our third-party driver authors notice and go thru the usual 3-cycle deprecation process. I don't even see so much as a mention in the driver changelog